# REHL does not complain if cd command has multiple arguments but the first is a directory, Debian does. 
# May 14th 2019 by B. Hajgato (Ghent University)
--- Nek5000-17.0/3rd_party/cvode/install.orig	2017-12-17 10:08:30.000000000 +0100
+++ Nek5000-17.0/3rd_party/cvode/install	2019-05-14 15:40:38.398947000 +0200
@@ -17,7 +17,7 @@
 
 tar -zxf *.tar.gz
 
-cd sundials*
+cd sundials-2.7.0
 mkdir -p build
 cd build
 
